frontend/src/utils/offlineMessages.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * These are used as fallback when the app is in offline mode. (no messages api fetched from the BE)
+ */
+module.exports = {
+  offline: {
+    de_DE: {
```

Comment:
Do we really need to be this specific with those languages ? can't just have `de` & `en` ? This would simplify the whole thing.